### PR TITLE
Fixed component help popup breaks in UI, bug #655

### PR DESCRIPTION
**Pull request by wadleo**
Hope this solves all the problem report at https://github.com/mit-cml/appinventor-sources/issues/655 and please test and reply, thanks.

### CODE REVIEW

**Comment by E-Hon** [comment]
Tested on OSX 10.11.4; Chrome 50.0.2661.86 (64-bit); Firefox 45.0.2; Safari 9.1 (11601.5.17.1)

The commit fixes issue #655. The bug cannot be reproduced anymore on the originally affected and other components.

Note that in Safari, the pop-up has a thin gap (see below) but it's not due to this commit. I don't know what causes it. 

gap

File: appinventor/appengine/war/Ya.css [hunk 1]
```diff
@@ -1030,6 +1030,11 @@ select.ode-PropertyEditor[disabled] {
   color: #8fc202;
 }
 
+.ode-ComponentHelpPopup > .popupContent > table{
+	width: 100%;
```
Comment (josmas) [inline appinventor/appengine/war/Ya.css]: please watch out for indentation. Should be 2 spaces (4 for continuation lines). Also note a space before the '{' in other css rules in this file.

**Comment by jisqyv** [comment]
@wadleo LGTM. Moving to Gerrit (I'll fix the indentation).

**Comment by jisqyv** [comment]
@wadleo MERGED into master

**Pull request closed without merging**